tests/test_packedtype.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+from dissect import cstruct
+from dissect.cstruct.types import PackedType
+
+from .utils import verify_compiled
+
```

Comment:
Can you add just a small `test_packedtype_float` that only does a couple `.reads()` and `.dumps`?